locale/fi/LC_MESSAGES/django.po

```diff
@@ -1131,48 +1132,52 @@ msgstr ""
 "Assistentin lisääminen epäonnistui, sillä käyttäjää opiskelijanumerolla {id} "
 "ei ole olemassa."
 
-#: edit_course/operations/configure.py:417
+#: edit_course/operations/configure.py:418
 msgid "Categories required as an object."
 msgstr "Kategoriat vaaditaan objektina."
 
-#: edit_course/operations/configure.py:420
+#: edit_course/operations/configure.py:421
 msgid "Modules required as an array."
 msgstr "Moduulit vaaditaan taulukkona."
 
-#: edit_course/operations/configure.py:428
+#: edit_course/operations/configure.py:429
 msgid "Category requires a name."
 msgstr "Kategoria vaatii nimen."
 
-#: edit_course/operations/configure.py:466
+#: edit_course/operations/configure.py:467
 msgid "Module requires a key."
 msgstr "Moduuli vaatii avaimen."
 
-#: edit_course/operations/configure.py:569
-msgid "ExerciseCollection object requires collection_category."
+#: edit_course/operations/configure.py:567
+#, fuzzy
+#| msgid "ExerciseCollection object requires collection_category."
+msgid "ExerciseCollection requires collection_category."
 msgstr "Tehtäväkokoelmaolio tarvitsee collection_category:n."
 
-#: edit_course/operations/configure.py:572
-msgid "ExerciseCollection object must have either identified or URL"
-msgstr "Tehtäväkokoelmaolion täytyy omata joko määrittely identified tai URL."
-
-#: edit_course/operations/configure.py:580
-msgid "Course: {} does not exist"
-msgstr "Kurssi: {} ei ole olemassa"
+#: edit_course/operations/configure.py:570
+#, fuzzy
+#| msgid "ExerciseCollection object requires collection_category."
+msgid "ExerciseCollection requires URL to target course instance."
+msgstr "Tehtäväkokoelmaolio tarvitsee collection_category:n."
 
-#: edit_course/operations/configure.py:586
-msgid "Course: {}, Instance: {}, not found."
-msgstr "Kurssi: {}, Instanssi: {}, ei löytynyt."
+#: edit_course/operations/configure.py:576
+msgid "Course URL '{}' doesn't match             service's hostname '{}'"
```

Comment:
Dman. These are actually from previous changes, which did not update translations. I'll try to update them and then you can rebase again, so we get rid of them here.